src/outputs/parthenon_hdf5.hpp

```diff
@@ -256,6 +256,28 @@ static void writeH5ASTRING(const char *name, const std::string &pData,
   PARTHENON_HDF5_CHECK(H5Awrite(attribute, atype, pData.c_str()));
 }
 
+static void writeH5ASTRINGS(const char *name, const std::vector<std::string> &pData,
+                            const hid_t &dSpace, const hid_t &dSet) {
+  int max_name_length = 0;
+  for (const auto &s : pData) {
+    max_name_length = (max_name_length > s.length()) ? max_name_length : s.length();
```

Comment:
Should use `std::max` here, but it appears that `max_name_length` doesn't get used. I wouldn't worry about this function, though, because I have a similar, more general, function to write vector attributes in my PR #486, so I'll remove this in that PR.